ui/js/services/search.js

```diff
@@ -92,14 +91,14 @@ class SearchDropdown {
 
             switch (e.which) {
                 case 13: // enter button 
-                    timetableUI.addCourseWithElement($('.search-dialog > div').eq(this.currentlySelected));
+                    searchDropdown.onChoose($('.search-dialog > div').eq(this.currentlySelected));
                     break;
                 case 38: // up
                     e.preventDefault();
-                    this.selectOptionWithIndex(-1);
+                    searchDropdown.selectOptionWithIndex(-1);
```

Comment:
Maybe consider up as 1 and down as - 1 for better readability

Reply:
@labib-c  I tried doing this but I reaized I can't (I think this is the reason why i made the up button send -1 and down +1 in the first place aswell). The reason is this: 

The initial value for the `currentlySelected` var is -1 because when it opens, nothing is selected. When the first option is selected (with the down arrow), the `currentlySelected` var becomes `0`... and it continues to advance as you go to the next option. When you're going back to the the original, the dropdown will go all the way back to -1. So when I tried implementing your suggestion, the selection broke because the down arrow makes it subtract 1 and the currentlySelected never goes up so it screws the rest of the search. I can't make currentlySelected be a positive number because the indexes in the search results array are all positive so we need to go in terms of that. It starts at -1 because there's never a possibility of -1 being an index in the array.

Reply:
ok I believe u, good effort